### PR TITLE
Update NeuralNet.py

### PR DESCRIPTION
**Pull request by manonreau**
Add the possibility to load an independent validation set

### CODE REVIEW

File: graphprot/NeuralNet.py [hunk 1]
```diff
@@ -22,16 +22,24 @@ class NeuralNet(object):
...
         # dataset
         dataset = HDF5DataSet(root='./', database=database, index=index,
                               node_feature=node_feature, edge_feature=edge_feature,
                               target=target)
+        PreCluster(dataset, method='mcl')
 
         train_dataset, valid_dataset = DivideDataSet(
             dataset, percent=percent)
 
+        if database_eval :
```
Comment (NicoRenaud) [inline graphprot/NeuralNet.py:36]: could you change the the condition to if `database_eval is not None` ? That would help readability of the code.

**Review by NicoRenaud — CHANGES_REQUESTED** [review]
Great that you added the valid database. Could you just change the condition in the if statement ?

**Comment by manonreau** [comment]
> Great that you added the valid database. Could you just change the condition in the if statement ?

done !

**Review by NicoRenaud — APPROVED** [review]
